### ISSUE TITLE
[content] Space missing from the organisations using Open Referral UK

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#174]
URL: 
https://openreferraluk.org/
https://openreferraluk.org/standard-community

Space is missing from the numbers of organisations using Open Referral UK and their status (considering, adopting, adopted). It currently reads: 
- 6considering
- 4adopting
- 6adopted

<img width="1260" alt="Screenshot 2021-06-23 at 16 40 40" src="https://user-images.githubusercontent.com/2459151/123127241-ff880880-d441-11eb-831f-0f5d98294163.png">

**Comment by UsmanAfzal** [comment]
Checked and approved
